.github/workflows/review.yml

```diff
@@ -9,6 +9,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - name: Install Go
```

Comment:
Added this to pass linter. Without this the linter will find standard library from Go 1.15.15, which misses something Filecoin requires.